pom.xml

```diff
@@ -99,6 +99,50 @@ The Initial Developer of the Covered Software is
         <groupId>org.sonatype.plugins</groupId>
         <artifactId>nexus-staging-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>4.1</version>
+        <configuration>
+          <defaultBasedir>${project.basedir}</defaultBasedir>
```

Comment:
@mathieucarbou I don't understand why I needed this.

Without it, `<exclude>**/target/**</exclude>` did not work, because the plugin was starting from `target/` (this is true at least for `terracotta/` submodule), and could not exclude the basedir itself.

Specifically, this was called with an empty name and target directory:
https://github.com/mathieucarbou/license-maven-plugin/blob/master/license-maven-plugin/src/main/java/com/mycila/maven/plugin/license/util/Selection.java#L90

Reply:
@mathieucarbou ping

Reply:
For the record:

Because shade plugin changes it:

https://maven.apache.org/plugins/maven-shade-plugin/shade-mojo.html#dependencyReducedPomLocation